templates/vue/src/components/tyde/TydeLog.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <li class="log">
+    <img :src="log.imageURL" />
+    <div class="log-details">
+      <div>{{log.title}}</div>
+      <div>{{log.description}}</div>
```

Comment:
Can you change this to use semantic tags? i.e. `h1` for titles and `p` for text/description. It makes it more obvious what's supposed to go there and it's better for accessibility in the future.

Reply:
Fixed this. Thanks.